src/components/dropdown/react/Dropdown.tsx

```diff
@@ -64,12 +85,63 @@ const DEFAULT_PROPS: IDefaultPropsType = {};
 const Dropdown: React.FC<DropdownProps> = ({
     children,
     className = '',
-    ...props
-}: DropdownProps): React.ReactElement => (
-    <div className={classNames(className, handleBasicClasses({ prefix: CLASSNAME }))} {...props}>
-        {children}
-    </div>
-);
+    closeOnClick = DEFAULT_PROPS.closeOnClick,
+    escapeClose = DEFAULT_PROPS.escapeClose,
+    offset,
+    // overToggle = DEFAULT_PROPS.overToggle,
+    position,
+    toggleElement,
+    width,
+}: DropdownProps): React.ReactElement => {
+    const wrapperRef: React.RefObject<HTMLDivElement> = useRef(null);
+    const [isOpen, setIsOpen]: [boolean, (isOpen: boolean) => void] = useState(Boolean(false));
```

Comment:
I think you can use `useState<boolean>(false)` instead

Reply:
ok - I just saw it the way I did it somewhere else in the project so replicated that. I'm fine with either tbh.